lib/rules/require-super-in-init.js

```diff
@@ -76,15 +76,29 @@ module.exports = {
       url:
         'https://github.com/ember-cli/eslint-plugin-ember/tree/master/docs/rules/require-super-in-init.md',
     },
-    fixable: null,
+    fixable: 'code',
     schema: [],
   },
 
   create(context) {
     const message = 'Call this._super(...arguments) in init hook';
 
     const report = function (node) {
-      context.report(node, message);
+      context.report({
+        node,
+        message,
+        fix(fixer) {
+          if (node.value.body.body.length > 0) {
+            // Function has at least one statement in it so just insert before that.
+            return fixer.insertTextBefore(node.value.body.body[0], 'this._super(...arguments);\n');
+          } else {
+            // Function is empty so insert after curly brace.
```

Comment:
If the function is empty, shouldn't we just remove it? Or will a separate rule do that?

Reply:
Not really since the user may be about to fill it in.